e2e/utils.go

```diff
@@ -572,6 +573,112 @@ func validatePVCClone(sourcePvcPath, clonePvcPath, clonePvcAppPath string, f *fr
 	validateRBDImageCount(f, 0)
 }
 
+func validateController(f *framework.Framework, pvcPath, appPath, scPath string) error {
```

Comment:
A comment describing the steps of the validation would be a great help! I am not sure if all the validation done here is intended/needed. Might be helpful to split certain things out in helper functions so that different parts of the validation are easier to understand/review.

Reply:
added comments for better understanding.